### DISCUSSION TITLE
post/slack-bot-improving-operational-efficiency/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#25] [Announcements]
# post/slack-bot-improving-operational-efficiency/

슬랙봇으로 업무 효율성과 마음 챙기기

https://tech.kakaopay.com/post/slack-bot-improving-operational-efficiency/

**Comment by heond913** [comment]
프로 작가 아리!!

**Comment by chero-kee** [comment]
고생 많으셨습니다!!

**Comment by smilejj91** [comment]
카카오페이의 배포 프로세스에 대해 잘 이해가 되었어요!

**Comment by dududumi** [comment]
좋은 글 감사합니다.

**Comment by ankonho** [comment]
이번 포스팅 너무 좋아요 :) 운영업무에 자동화로 미치는 업무 질의 향상 너무 잘봤습니다

**Comment by unme88** [comment]
의문에 그치지 않고 실행까지 옮긴 게 인상 깊네요👍👍

**Comment by yus1108** [comment]
고생하셨어요!! 저도 업무를 진행하다보면 자동화할수 없을까에 대해 고민하게 되는데 많은 참고가 되었습니다~

**Comment by harryplusplus** [comment]
같은 하늘의 달을 보더라도 모두가 다른 생각을 하는 것 처럼, 같은 현상을 보더라도 어떤 이에게는 사람이 ”잘“ 해야하는 영역인 것이고, 어떤 이에게는 업무 자동화의 영역으로 보이는 것 같다.

문제를 명료하게 정의하는 것은 문제를 이미 해결한 것과 같다고 할 수 있다.

**Comment by HyeJinP** [comment]
정말 유익한 글입니다. 저도 자동화에 관심이 있었는데, 이번 기회에 더 깊이 알아간 것 같아요 :)
